<ui version="4.0" >
 <class>GoToPageWidget</class>
 <widget class="QDialog" name="GoToPageWidget" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>68</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Go to page...</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../data/qcomicbook.qrc" >
    <normaloff>:/icons/qcomicbook.png</normaloff>:/icons/qcomicbook.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Page number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_pagenumber" />
        </item>
        <item>
         <widget class="QLabel" name="lb_total" >
          <property name="text" >
           <string>/0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_go" >
          <property name="text" >
           <string>Go</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../data/qcomicbook.qrc" />
 </resources>
 <connections/>
</ui>
